src/Services/CodeParser/CodeParser.php

```diff
@@ -127,9 +129,13 @@ public function getMethodCalls(
     public function getFunctionCalls(string $functionName): array
     {
         $calls = $this->nodeFinder->find($this->nodes, function (Node $node) use ($functionName) {
-            return $node instanceof Expression &&
-                $node->expr instanceof FuncCall &&
-                $node->expr->name->toString() === $functionName;
+            try{
+                return $node instanceof Expression &&
+                    $node->expr instanceof FuncCall &&
+                    $node->expr->name->toString() === $functionName;
+            } catch (Throwable) {
+                return false;
+            }
```

Comment:
I don't like using a try-catch to silently ignore errors, I'd rather fix the actual issue it's throwing on.

The following suggestion reverts the try-catch and instead make sure the node is a `\PhpParser\Node\Name` before trying to call `toString` on it.

I tried this with the test case you added and it seems to work, WDYT?

```suggestion
            return $node instanceof Expression &&
                $node->expr instanceof FuncCall &&
                $node->expr->name instanceof \PhpParser\Node\Name &&
                $node->expr->name->toString() === $functionName;
```

Reply:
> I don't like using a try-catch to silently ignore errors, I'd rather fix the actual issue it's throwing on.
> 
> The following suggestion reverts the try-catch and instead make sure the node is a `\PhpParser\Node\Name` before trying to call `toString` on it.
> 
> I tried this with the test case you added and it seems to work, WDYT?

Perfect. 

I think you have better knowledge here in this topic, I did not even think of this approach 😅